.rubocop_todo.yml

```diff
@@ -1,11 +1,18 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2024-06-28 12:36:37 UTC using RuboCop version 1.64.1.
+# on 2025-02-25 12:00:05 UTC using RuboCop version 1.64.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
+# Configuration parameters: Severity, Include.
+# Include: **/*.gemspec
+Gemspec/RequiredRubyVersion:
+  Exclude:
+    - 'puppetlabs_spec_helper.gemspec'
```

Comment:
I'd consider this poor practice. You are telling RuboCop to assume you can use Ruby 2.7 features while being Ruby 2.6 compatible. It's just a matter of time before that breaks IMHO.

Reply:
Yes. I think the resource-api needs to drop Ruby 2.6 support

Reply:
I think you misunderstood the issue. IIRC the real issue is that JRuby used in Puppetserver 7 is officially only Ruby 2.6 compatible so I'm guessing some of Perforce's internal CI jobs run with Ruby 2.6.

Reply:
Yes. But that's EoL in three days: https://www.puppet.com/products/puppet-enterprise/support-lifecycle. So why does resource-api *now* need to use latest puppetlabs_spec_helper + Ruby 2.6?

Reply:
That's something I can't answer.